php-console.js

```diff
@@ -181,6 +181,53 @@
                 e.preventDefault();
             });
         }
+        
+        //Save
+        if (window.localStorage) {
```

Comment:
I think we can rely on localStorage being there these days.. especially for such a dev oriented tool.